src/utils/taskUtil.js

```diff
@@ -0,0 +1,38 @@
+
+
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Too many blank lines at the beginning of file. Max of 1 allowed.](https://app.codacy.com/gh/Strooped/strooped-master/pullRequest?prid=5420986)